src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -243,6 +246,9 @@
   <data name="MismatchedObjectArray" xml:space="preserve">
     <value>'{0}' is invalid without a matching open.</value>
   </data>
+  <data name="CannotWriteEndAfterProperty" xml:space="preserve">
+    <value>'{0}' is invalid right after a property name.</value>
```

Comment:
saying "right after" is a little bit informal, what you think of changing this to something like `'{0}' is invalid immediately following a property name.` or `'{0}' is invalid following a property name.`

Reply:
I prefer `'{0}' is invalid following a property name.`